index.html

```diff
@@ -679,19 +678,19 @@ <h4>NewPipe is free and open source</h4>
                     <h3 class="title">Support NewPipe</h3>
                 </div>
                 <div class="col-md-6 tile tile-left" id="tile-free">
-                    <h4>Why NewPipe is free</h4>
+                    <h4>Why NewPipe is libre and gratis</h4>
                     <ul class="check">
                         <li class="heart">We love what we are doing</li>
-                        <li>NewPipe is powered by a growing community</li>
-                        <li>Our goal is to make the internet a more free (libre) place and open it for everyone</li>
-                        <li>Your donations cover a majority of our costs</li>
-                        <li class="text-success">F-Droid, Weblate, Sentry and other companies offer their services to open source projects for free</li>
+                        <li>Powered by a growing and generous community</li>
+                        <li>To make the Internet for everyone</li>
```

Comment:
```suggestion
                        <li>To make the Internet more accessible for everyone</li>
```

Reply:
I thought "to make the Internet for everyone" is an OK slogan/saying. NewPipe should be better for blind users before the accessible for everyone holds true. I think the brevity encompasses everyone nicely?

Reply:
To make the internet _____ for everyone. It's missing an adjective.

Reply:
Was going for "To make _the Internet for everyone_"

Reply:
Hmmm. To me it reads like Newpipe is creating this thing called the Internet for everyone.

Reply:
maybe focus on the media part, what NewPipe is about?

To make media streaming for everyone?

Reply:
We already know what it is about from earlier, this is a greater goal. To its greatest extent, it is summarized in those 8 words. Improvements would be, fewer words, better sounding sentence, or similar.

Reply:
To make the Internet a better place for everyone?